frontend/src/router/index.js

```diff
@@ -0,0 +1,23 @@
+import { createRouter, createWebHistory } from 'vue-router';
```

Comment:
Could you maybe create routes for all major pages that are expected? It will  create a structure and be a huge help. 
We definitely need the page something like `/searchavolunteer`, `registration`, also profile page and volunteer's page to contact (but that needs to be connected to backend) ...
And `/thankyou` :)

Reply:
yes, sure, some empty files are already there, but I will create more and also write the connections in the router